tests/dom/global.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* global window, document */
+
+import { window as globWindow, document as globDocument } from 'ckeditor5/utils/dom/global.js';
```

Comment:
I hope bundlers will handle it properly. Otherwise, it's a simple matter of `const nativeDomWindow = window` before importing and then `expect( window ).to.equal( nativeDomWindow );`